usaon_benefit_tool/templates/home.html

```diff
@@ -2,38 +2,7 @@
 
 
 {% block content %}
-  <!--Thinking about how this should be due to the way that it links to Benefit Tool docs -->
-  <h2>{% block title %}Welcome to the US AON Benefit Tool!{% endblock %}</h2>
-    <p><strong>Actions:</strong></p>
-    <ul>
-      <li>View our <a href="{{ url_for("surveys.view_surveys") }}">library of evaluations here.</a></li>
-      <li>Click here to <a href="{{ url_for("surveys.view_surveys") }}">edit or update your own evaluation.</a></li>
-      <li>If you'd like to create a new evaluation please email: <a href="mailto:hazel@iarpccollaborations.org">hazel@iarpccollaborations.org</a> </li>
-      <li>Our <a href="https://docs.google.com/document/d/13DLX3A__M60xMgFbQqk2NMAjZzahn5FZcvB3fWSVRck/edit?usp=sharing">data management plan</a> includes information about how any information you add to the tool will be stored and may be shared. Please review the data management plan here.</li>
-      <li><a href="https://usaon.org/evaluation-and-planning/benefit-tool">Learn more about our Benefit Tool.</a></li>
-    </ul>  
- 
-<p><strong>Introduction</strong></p>  
-<p>The Benefit Tool illustrates links between observing systems, data products, 
-      and applications and shows how they connect to societal benefit. Capabilities 
-      and gaps are given numerical ratings and written descriptions throughout the evaluation. </p> 
- 
-<p>Ratings are given for a particular context by an individual or small team of experts 
-      (called respondents). They are subjective but well-informed representations of reality. 
-      The ratings convey the input of the respondents, not US AON's assessment of a given 
-      product or system. </p> 
- 
-<p>Societal benefit ratings present a particular challenge because benefits are context-dependent
-      and often not quantifiable. Like other Western science evaluation frameworks, the Benefit 
-      Tool struggles to depict this nuance. US AON is moving towards cohort ratings for societal 
-      benefit, in line with recommendations coming out of the Arctic Observing Summit  and the 
-      US AON Expert Committee on Methods. However, the current version of the tool includes 
-      evaluations with more limited assessments of societal benefits. </p> 
- 
-<p>Individual evaluations can show the importance of critical capabilities -- be that the strength 
-      of a long-term observing system, the value of a robust communication system, the impact of 
-      data product improvements, or more -- or the impact of specific gaps. As more people use 
-      the Benefit Tool, the value of the evaluations grows. We will begin to see more systemic 
-      strengths and weaknesses in the Arctic observing system, and opportunities to invest in systems 
-      that will have wide-reaching impacts. </p>
+  {% filter markdown %}
+    {% include "home.md" %}
+  {% endfilter %}
```

Comment:
These three lines are what we will need any time we want to inject markdown in to a template. The filter `markdown` is established as part of the change to `__init__` above.